test_e2e.sh

```diff
@@ -295,6 +295,7 @@ function GIT_SYNC() {
         -v "$DOT_SSH/1/id_test":"/ssh/secret.1":ro \
         -v "$DOT_SSH/2/id_test":"/ssh/secret.2":ro \
         -v "$DOT_SSH/3/id_test":"/ssh/secret.3":ro \
+	-v "$(pwd)/$GITHUB_APP_PRIVATE_KEY_FILE":"/github_app_private_key.pem":ro \
```

Comment:
This requires the github app private key file to be located at the same directory. It won't work with the e2e test in the GitHub Actions Workflow. 

I'm not sure if you get a chance to take a look at https://github.com/risset/git-sync/pull/1/commits/f02d659803e20a00f988d54c91e529dbd7daf9ca. Basically, it loads the private key from GitHub Secrets and stored it in a temp file. It would be good to incorporate it in this PR and enables the e2e test.

Reply:
I will take a look at that

Reply:
Thanks. I updated https://github.com/risset/git-sync/pull/1 to make the github app auth test skippable.

Reply:
Merged this in now